CONTRIBUTING.md

```diff
@@ -512,6 +520,22 @@ into any of those categories, use your best judgement.
 * Use `HasX` for protocols that have readable and/or writable attributes
   or getter/setter methods (e.g. `HasItems`, `HasFileno`).
 
+### `@deprecated`
+
+Typeshed uses the `@typing_extensions.deprecated` decorator
+(`@warnings.deprecated` since Python 3.13) to mark deprecated
+functionality; see [PEP 702](https://peps.python.org/pep-0702/).
+
+A few guidelines for how to use it:
+
+* In the standard library, apply the decorator only in Python versions
```

Comment:
Commented in the issue, but I think it would be nice to be more aggressive in cases where an appropriate replacement already exists

Reply:
I agree now, reworded the section to say we should apply it to versions where a good replacement exists.